packages/design-system/src/components/N8nUserInfo/UserInfo.vue

```diff
@@ -51,6 +51,16 @@ export default mixins(Locale).extend({
 		isCurrentUser: {
 			type: Boolean,
 		},
+		disabled: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	computed: {
+		classes(): string[] {
+			return [this.$style.container]
+				.concat(this.disabled ? [this.$style.disabled] : []);
```

Comment:
nitpick - I fine object style `{[this.$style.container]: true, [this.$style.disabled]: this.disabled}` to be a little clearer

Reply:
Done! Good idea.